src/frontend/efiling-frontend/src/App.js

```diff
@@ -1,19 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
 import { Switch, Route, Redirect, useHistory } from "react-router-dom";
 import Home from "./components/page/home/Home";
 
+// For cancel button and confirmation popup
+const mainButton = {
```

Comment:
these red bars on the margin apparently mean that we're missing test coverage on these...somehow?

Reply:
shouldnt need test coverage for app.js... I thought we were ignoring this, not sure why it is reporting it

Reply:
yea no clue either, was merely pointing it out for transparency